### PR TITLE
Done with my US

### PR DESCRIPTION
**Pull request by shuseinzoda**
Hey guys I am done with my US. Kindly ask you to approve merge request. Let me know if you want to add something.

### CODE REVIEW

**Review by Razia1975 — APPROVED** [review]
Good job!

File: src/test/java/com/symund/pages/FilesPage.java [hunk 1]
```diff
@@ -22,16 +56,33 @@ public static WebElement selectTab(String tabName) {
             return null; // Return null if the module is not found.
         }
 
-
         List<WebElement> tabs = header.findElements(By.tagName("a"));
         for (WebElement eachTab : tabs) {
-            if (eachTab.getText().contains(tabName.substring(0,1).toUpperCase())){ // will change the 1st letter of the tabName to UpperCase
+            if (eachTab.getText().contains(tabName.substring(0, 1).toUpperCase())) { // will change the 1st letter of the tabName to UpperCase
```
Comment (ElenaUS030) [inline src/test/java/com/symund/pages/FilesPage.java:61]: line 61 in your code is slightly different from line 28 from old code (has extra spaces)

File: src/test/java/com/symund/pages/FilesPage.java [hunk 2]
```diff
@@ -2,17 +2,51 @@
...
+import org.openqa.selenium.support.FindBy;
 import org.openqa.selenium.support.PageFactory;
 
+
 import java.util.List;
 
 public class FilesPage {
 
-    public FilesPage(){
+    public FilesPage() {
```
Comment (ElenaUS030) [inline src/test/java/com/symund/pages/FilesPage.java:24]: line 24 in your code is slightly different from line 13 from old code (has extra space)

**Review by ElenaUS030 — APPROVED** [review]
Looks good! Here are couple of things to correct:
1. Two lines have slight difference (extra spaces) but do not affect the code in general.
2. In scenarios you have Deleted files tab and Deleted files module - same folder, different names. Maybe rename them in one style and make changes in the code as well.
All in all - great job!

**Pull request merged**